src/main/java/seedu/java/util/Parser.java

```diff
@@ -0,0 +1,74 @@
+package seedu.java.util;
+
+/**
+ * All static method. Handles input from user.
+ * readCommand(String)
+ * readTask(String)
+ * readTiming(String)
+ * readNum(String)
+ */
+public class Parser {
+    /**
+     * Static method. Returns a Command (inferred instruction)
+     * @param input from UI
+     * @return Command
+     * @throws Exception
+     */
+    public static Command readCommand(String input) throws Exception {
```

Comment:
I think that the javadocs for this method can be clearer, e.g. why is the Exception being thrown? Also, adding on to @alcen, it would be clearer to perhaps create a custom exception like InvalidCommandException.